2_segmenting_data/2.segment_NF1_data.sh

```diff
@@ -0,0 +1,4 @@
+
+#!/bin/bash
+jupyter nbconvert --to python Segmentation_Pipeline.ipynb
+python Segmentation_Pipeline.py
```

Comment:
```suggestion
python Segmentation_Pipeline.py

```

Remember to add new lines to the ends of files to ignore that red symbol!

![image](https://user-images.githubusercontent.com/7340421/185419004-fa042955-e4f4-41c4-8ad1-12c85759f7db.png)

Reply:
Done!